invenio_vocabularies/records/models.py

```diff
@@ -56,22 +56,22 @@ class VocabularyMetadata(db.Model, RecordMetadataBase):
 
 
 class VocabularySubtype(db.Model):
-    """Vocabulary subtype type model.
+    """Vocabulary subtype model.
 
-    Current use case is for subject types.
+    This is used to store metadata for subject schemes for now.
+
+    Since we might support different affiliations schemes and different funding
+    schemes, this table is generic. Hopefully it is not overeach.
     """
 
     __tablename__ = "vocabularies_subtypes"
 
     id = db.Column(db.String, primary_key=True)
-    vocabulary_id = db.Column(
-        db.String,
-        db.ForeignKey(VocabularyType.id, ondelete='CASCADE'),
-        nullable=False,
-    )
-    label = db.Column(db.String)
-    prefix_url = db.Column(db.String)
-    """Any extra metadata is added as columns."""
+    # This is e.g. `subjects`, 'affiliations', ...
+    parent_id = db.Column(db.String, primary_key=True)
```

Comment:
Should it be an FK? or not because there wont be an entry in the `VocabularyType` table?

Reply:
Not a FK because no entry in VocabularyType table for specific vocabularies.